coralproject/talk
# Fix: CORL-2480 open story drawer

## What does this PR do?
Removes the StoryActions dropdown and replaces uses that button to open the StoryInfoDrawer, removes "Open in new" icon in StoryRow

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
Go to Admin > Stories, see that the "Open in. new" icon is gone. Click the three dots that used to open StoryActions and see that the Story Info Drawer opens
